tensorboard/backend/application_test.py

```diff
@@ -186,55 +189,54 @@ class TensorboardServerUsingMetagraphOnlyTest(TensorboardServerTest):
 
 class ParseEventFilesSpecTest(tf.test.TestCase):
 
+  def customFsUtil(self, pathObj, logdir, expected):
```

Comment:
`customFsUtil` is a strange name. This is effectively a custom assert, so maybe:
`assertPlatformSpecificLogdirParsing`
with a docstring that explains what pathObj is